### ISSUE TITLE
Suggestions for RuneLite epicness

### ISSUE THREAD

**Issue by Boreall13** [runelite/runelite#3622]
My first suggestion is adding prices for potential sale value on items. For example, Dragonfire shields are untradable while they have charges in them, so they are counted as a 0 in bank value. This also goes for items such as abyssal whips which turn tentacles when expired. 
To try and put it concisely, try to add values to items which can be sold at any point, not just currently.

I'd also like a way to merge HA/Wildy values to items that are untradeable (like Torso)

Different text options for ground text would be great as well. Currently it looks a little blocky an sharp, and can be hard to read.

I'm sure this one has been said many times, but a way to use !Kc and !Task would be amazing as well. 

Thank you all for the hard work you do to give us an incredible client

**Comment by GETrackerDan** [comment]
I've finished !task will PR shortly.

**Comment by Reasel** [comment]
> Different text options for ground text would be great as well. Currently it looks a little blocky an sharp, and can be hard to read.

What do you mean by this?

**Comment by Boreall13** [comment]
@Reasel Maybe it's just me but compared to regular runescape text (I guess I don't mean blocky) It just feels like it's too small, and sharp. I had to turn off ground prices because it genuinely hurt my eyes to try and read it. 

Also, the triforce gets instalove from me so <3

**Comment by Reasel** [comment]
@Boreall13 I ask because I didn't like it either so I added a bold font that perfectly matches the right click font. Its under RuneLite settings called Dynamic Overlay Font -> Bold

**Comment by Boreall13** [comment]
@Reasel Am I crazy for not finding this?

**Comment by Reasel** [comment]
No, its a bit hidden. It makes sense since it changes all overlays that RuneLite uses. It would be strange if each plugin had its own.

**Comment by Boreall13** [comment]
@Reasel I just found it, you are a saint. You get double love

**Comment by Boreall13** [comment]
@deathbeam Some items like Torso, Fire Cape, etc (Most untradeables) Have either a high alchemy value attached to them, or drop X amount of gold when dropped in the Wildy, those values would be nice to see in your total bank value

**Comment by deathbeam** [comment]
I believe they are already included in the HA value.

**Comment by Boreall13** [comment]
@Deathbeam sorry I guess I'm not explaining well. 

I meant an option to merge HA Value and GE value into one number

**Comment by GETrackerDan** [comment]
What is the benefit of combining high alch value with grand exchange price value?

**Comment by Boreall13** [comment]
@GETrackerDan the benefit is that some people prefer it? Isn't that what RuneLite is all about? A client made the way players want?

**Comment by GETrackerDan** [comment]
@Boreall13 get off your high horse I'm asking a question as to what the actual benefit for doing this is as I'm aware of one. If I wanted to know the price of an item I have it. If I need the high alch price I have it but why both combined?

Why would you prefer it? Explain.

**Comment by williammetcalf** [comment]
A possible benefit I can think of is an ironman wanting to know how much their alch tab would alch for, but since most of the GE price for these items are alch price anyways.... not sure how necessary it really is

**Comment by Boreall13** [comment]
@williammetcalf i would like to see the total value of my bank, everything sellable, alchable, or potentially sellable (such as charged weapons/shields) under a single number

**Comment by Nightfirecat** [comment]
For the issue title (that is, "Add bank value for abyssal tentancle and untradeables converted to coins on death"), hasn't this issue been solved?
